libs/styles/src/lib/scss/_apply.scss

```diff
@@ -114,6 +114,16 @@
   @include cvi-form-message;
 }
 
+.cvi-table-responsive {
+  @include cvi-table-responsive;
+  @include cvi-table;
+  @include cvi-card;
+}
+
+.cvi-card {
```

Comment:
It might make sense to change this name to less generic (eg `data-card` or something similar). We are likely to have more card components in future.

Reply:
Refactored name as table-card. As it is used in table responsive.